plugin/health/health.go

```diff
@@ -28,16 +30,25 @@ type health struct {
 }
 
 // newHealth returns a new initialized health.
-func newHealth(addr string) *health {
-	return &health{Addr: addr, stop: make(chan bool), pollstop: make(chan bool)}
+func newHealth(alloc listener.AllocationToken) *health {
+	return &health{alloc: alloc, stop: make(chan bool), pollstop: make(chan bool)}
 }
 
-func (h *health) OnStartup() error {
-	if h.Addr == "" {
-		h.Addr = defAddr
+func (h *health) Tag() string {
```

Comment:
Can't we reuse Name() for this?

Reply:
However .. if you think it is preferable, I can change.

Reply:
I see you point, but right now it is exactly like Name()... (let me review further, but I think we should re-use name)